Clippit/PowerPoint/PresentationBuilderTools.cs

```diff
@@ -106,48 +106,54 @@ internal static void CopyChartObjects(ChartPart oldChart, ChartPart newChart)
             {
                 var relId = dataReference.Attribute(R.id).Value;
 
-                if (oldChart.Parts.FirstOrDefault(p => p.RelationshipId == relId) is {} oldPartIdPair)
+                if (!string.IsNullOrEmpty(oldChart.Parts.FirstOrDefault(p => p.RelationshipId == relId).RelationshipId))
                 {
-                    switch (oldPartIdPair.OpenXmlPart)
+                    if (oldChart.Parts.FirstOrDefault(p => p.RelationshipId == relId) is { } oldPartIdPair)
```

Comment:
let's extract `oldChart.Parts.FirstOrDefault(p => p.RelationshipId == relId)` into variable and combine 2 if conditions into one